src/main/java/Bot.java

```diff
@@ -0,0 +1,162 @@
+import java.io.PrintWriter;
+import java.util.Arrays;
+import java.util.LinkedList;
+
+public class Bot {
+    private final PrintWriter pw = new PrintWriter(System.out);
+    private static final String SEPARATOR = "__________________________________________________________________\n";
+    private static final String INDENTATION = "    ";
+    private final LinkedList<Task> list = new LinkedList<Task>();
+
+    public Bot(){}
+
+    public void greet() {
+        this.pw.print(INDENTATION + SEPARATOR
+                + INDENTATION + "Hello, Notnow here!\n"
+                + INDENTATION + "What can I do for you?\n"
+                + INDENTATION + SEPARATOR);
+        this.pw.flush();
+    }
+
+    public void exit() {
+        this.pw.print(INDENTATION + SEPARATOR
+                + INDENTATION + "Bye. Hope to see u again soon!\n"
+                + INDENTATION + SEPARATOR);
+        this.pw.flush();
+    }
+
+    public void consume(String input) {
+        String[] tokens = input.split(" ");
+        String taskType = tokens[0];
+        String content = String.join(" ", Arrays.copyOfRange(tokens, 1, tokens.length));
+        int index;
+
+        switch (taskType) {
+            default:
```

Comment:
Same comment as @belligerentbeagle, CS2103 requires that the case to be aligned with the switch. Please refer: https://se-education.org/guides/conventions/java/intermediate.html for " The switch statement should have the following form: Note there is no indentation for case clauses." section